source/renderer/app/stores/VotingStore.ts

```diff
@@ -89,6 +88,15 @@ export default class VotingStore extends Store {
     votingActions.closeConfirmationDialog.listen(this._closeConfirmationDialog);
 
     this._initializeFundPhaseInterval();
+
+    // @ts-ignore ts-migrate(1320) FIXME: Type of 'await' operand must either be a valid pro... Remove this comment to see the full error message
+    await this.getCatalystFundRequest.execute();
```

Comment:
This could be moved to a setup function as the `setup` store doesn't really wait for async operations to complete before bootstrapping the app. 

Also, would be possible to resolve the `ts-ignore`?

Reply:
Good catch. I'll update, thanks!